rust/tools/authorization-logic/build.rs

```diff
@@ -22,13 +22,13 @@ use std::{env, error::Error, process::Command};
 fn run_antlr_on(grammar_file: &str, antlr_path: &str) -> Result<(), Box<dyn Error>> {
     let parsing_subdir = env::current_dir().unwrap().join("src/parsing");
     let full_file = grammar_file.to_owned() + ".g4";
-
+    let java_bin = env::var("JAVA").unwrap_or("java".to_string());
```

Comment:
Why the `to_string` here? Isn't `"java"` already a string?

Reply:
Either the type checker or the borrow checker insisted that `to_string` is needed.